src/integrationTest/resources/simulations/__files/prisoner-A1234BC.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
The changes to the controller methods (see below) now need to get the prisoner from the prisoner-search-api, so I needed to create a stub 👍